app/controllers/sign_up/select_email_controller.rb

```diff
@@ -55,6 +55,16 @@ def last_email
       end
     end
 
+    def selected_email_id
+      if current_sp.present? &&
+         current_sp.attribute_bundle&.include?('all_email') &&
```

Comment:
There is no attribute `all_email`. This should be `all_emails`.

https://developers.login.gov/attributes/